one/src/events.rs

```diff
@@ -28,9 +35,25 @@ pub struct SlurpOpts {
     output_ceramic_path: Option<PathBuf>,
 }
 
+#[derive(Args, Debug)]
+pub struct ValidateOpts {
+    /// Path to storage directory
+    #[arg(short, long, env = "CERAMIC_ONE_STORE_DIR")]
+    store_dir: Option<PathBuf>,
+
+    /// CID of the block to validate
```

Comment:
of the block, or the transaction?

Can we call this arg something more descriptive than `cid` so it's clearer what it actually represents?  I have a bit of a pet peeve against variables with the same name as their type, it's just a missed opportunity to convey more information to the reader.

Reply:
I'll rename as `event`